src/components/Header/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import {
+  routesBottom,
+  routesMiddleAbove,
+  routesMiddleBottom,
```

Comment:
it is not routes 
change name to navigationTop navigationBottom and navigationMiddle for example